app/page.tsx

```diff
@@ -1,17 +1,11 @@
-"use client";
 import React from "react";
-import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
-import SearchBar from "@/components/SearchBar";
 import SearchResult from "../components/SearchResult";
```

Comment:
```suggestion
import Search from "../components/Search";
```

you're default exported component here is now `Search`, so I'd change the file name and the default export name where you're using it